Makefile

```diff
@@ -260,6 +260,23 @@ namespace:
 	@echo "Creating '$(NAMESPACE)' namespace/project"
 	@oc apply -f deploy/ns.yaml
 
+.PHONY: deploy
+deploy: namespace deploy-crds ## Deploy the operator from the manifests in the deploy/ directory
+	@oc apply -n $(NAMESPACE) -f deploy/
+
+.PHONY: deploy-local
+deploy-local: namespace image-to-cluster deploy-crds ## Deploy the operator from the manifests in the deploy/ directory and the images from a local build
+	@sed -i 's%$(IMAGE_REPO)/$(APP_NAME):latest%$(OPERATOR_IMAGE_PATH)%' deploy/operator.yaml
+	@oc apply -n $(NAMESPACE) -f deploy/
+	@sed -i 's%$(OPERATOR_IMAGE_PATH)%$(IMAGE_REPO)/$(OPERATOR_IMAGE_NAME):latest%' deploy/operator.yaml
```

Comment:
Sorry I didn't catch it the first time around, but there seems to be a typo, can you squash in this change?
```
diff --git a/Makefile b/Makefile
index beab6a0..3dcbba1 100644
--- a/Makefile
+++ b/Makefile
@@ -268,7 +268,7 @@ deploy: namespace deploy-crds ## Deploy the operator from the manifests in the d
 deploy-local: namespace image-to-cluster deploy-crds ## Deploy the operator from the manifests in the deploy/ direct>
        @sed -i 's%$(IMAGE_REPO)/$(APP_NAME):latest%$(OPERATOR_IMAGE_PATH)%' deploy/operator.yaml
        @oc apply -n $(NAMESPACE) -f deploy/
-       @sed -i 's%$(OPERATOR_IMAGE_PATH)%$(IMAGE_REPO)/$(OPERATOR_IMAGE_NAME):latest%' deploy/operator.yaml
+       @sed -i 's%$(OPERATOR_IMAGE_PATH)%$(IMAGE_REPO)/$(APP_NAME):latest%' deploy/operator.yaml
 
 .PHONY: deploy-local
 deploy-crds:
```